src/main/kotlin/com/group/libraryapp/service/user/FailTest.kt

```diff
@@ -4,8 +4,8 @@ class FailTest {
 
 
 
-    fun testtest( a:Int) {}
+    fun testtest22( a:Int) {}
```

Comment:
⚠️ **[detekt]** <detekt.UnusedPrivateMember> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function parameter `a` is unused.